Docs: rewrite recommended deployment for v6 (#2920)
nuts-foundation/nuts-node: docs/pages/integrating/api-authentication.rst

When authentication fails the API will return ``HTTP 401 Unauthorized``. The logs on the nuts-node will provide
an explanation about the failure.

.. note::

    The JWTs and private keys used in this authentication scheme are secrets and should never be shared with anyone. No one should ever ask you to send them your JWTs or private keys.

Configuration
-------------
Authentication can be enabled by setting ``http.internal.auth.type`` (see example above) to ``token_v2``.
Endpoints under ``/internal`` will then require a JWT signed by an authorized key.

> Authorized (public) keys are configured in an ``authorized_keys`` file as specified in the ``http.internal.auth.type.authorizedkeyspath`` parameter.

Reviewer: ```suggestion
Authorized (public) keys are configured in an ``authorized_keys`` file configure by the ``http.internal.auth.type.authorizedkeyspath`` parameter.
```
